src/ParameterStoreBuildWrapperPlugin.groovy

```diff
@@ -26,6 +32,11 @@ class ParameterStoreBuildWrapperPlugin implements TerraformEnvironmentStagePlugi
 
         stage.decorate(PLAN, addParameterStoreBuildWrapper(options))
         stage.decorate(APPLY, addParameterStoreBuildWrapper(options))
+
+        globalParameterOptions.each { gp ->
+            stage.decorate(PLAN, addParameterStoreBuildWrapper(gp))
+            stage.decorate(APPLY, addParameterStoreBuildWrapper(gp))
+        }
```

Comment:
Looks like code coverage went down. You modified the apply method, but there are no unit tests exercising that. Can you add a unit test that exercises the new code that you added? The new unit tests should run apply against a Stage, and should have global parameter paths, so that we exercise the loop you added at least once.

Reply:
I will